### PR TITLE
Update Startup2.cs

### PR DESCRIPTION
**Pull request by garyray-k**
Being more explicit with the naming of these variables allows for readers of the code to easily digest what the variable actually is and what it's purpose may be.

Related
#17504

### CODE REVIEW

File: aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs [hunk 1]
```diff
@@ -18,9 +18,9 @@ public void ConfigureServices(IServiceCollection services)
                 .SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
 
             // Register the Swagger generator, defining 1 or more Swagger documents
-            services.AddSwaggerGen(c =>
+            services.AddSwaggerGen(swaggerGeneratorOptions =>
```
Comment (scottaddie) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs]:
```suggestion
            services.AddSwaggerGen(options =>
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs [hunk 2]
```diff
@@ -18,9 +18,9 @@ public void ConfigureServices(IServiceCollection services)
...
             {
-                c.SwaggerDoc("v1", new OpenApiInfo { Title = "My API", Version = "v1" });
+                swaggerGeneratorOptions.SwaggerDoc("v1", new OpenApiInfo { Title = "My API", Version = "v1" });
```
Comment (scottaddie) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs]:
```suggestion
                options.SwaggerDoc("v1", new OpenApiInfo { Title = "My API", Version = "v1" });
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs [hunk 3]
```diff
@@ -33,9 +33,9 @@ public void Configure(IApplicationBuilder app)
 
             // Enable middleware to serve swagger-ui (HTML, JS, CSS, etc.),
             // specifying the Swagger JSON endpoint.
-            app.UseSwaggerUI(c =>
+            app.UseSwaggerUI(swaggerUIOptions =>
```
Comment (scottaddie) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs]:
```suggestion
            app.UseSwaggerUI(options =>
```

File: aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs [hunk 4]
```diff
@@ -33,9 +33,9 @@ public void Configure(IApplicationBuilder app)
...
             {
-                c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
+                swaggerUIOptions.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
```
Comment (scottaddie) [inline aspnetcore/tutorials/web-api-help-pages-using-swagger/samples/2.1/TodoApi.Swashbuckle/Startup2.cs]:
```suggestion
                options.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
```

**Comment by Rick-Anderson** [comment]
@garyray-k  can you roll your 3 PR's into one PR?

**Comment by garyray-k** [comment]
I'll roll them all into one PR.

**Comment by garyray-k** [comment]
Merging several PRs into one.

**Comment by Rick-Anderson** [comment]
@garyray-k  Thanks!

**Comment by Rick-Anderson** [comment]
And go with @scottaddie  suggestion and use `options`.

**Pull request closed without merging**